<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectionCloset</class>
 <widget class="QMainWindow" name="CollectionCloset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Коллекционный шкаф</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_insert">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Constantia</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Добавить..</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_update">
        <property name="font">
         <font>
          <family>Constantia</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Обновить..</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_delete">
        <property name="font">
         <font>
          <family>Constantia</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Удалить..</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="l_sort">
        <property name="font">
         <font>
          <family>Constantia</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Сортировать:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_sort">
        <property name="font">
         <font>
          <family>Constantia</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="a_change_user"/>
    <addaction name="separator"/>
    <addaction name="a_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <action name="a_add">
   <property name="text">
    <string>Добавить..</string>
   </property>
  </action>
  <action name="a_update">
   <property name="text">
    <string>Обновить..</string>
   </property>
  </action>
  <action name="a_delete">
   <property name="text">
    <string>Удалить..</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Отображение</string>
   </property>
  </action>
  <action name="a_change_user">
   <property name="text">
    <string>Сменить пользователя...</string>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="a_exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>а</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
